src/vivarium_public_health/metrics/mortality.py

```diff
@@ -142,3 +150,22 @@ def count_deaths(self, x: pd.DataFrame) -> float:
     def calculate_ylls(self, x: pd.DataFrame) -> float:
         died_of_cause = x["exit_time"] > self.clock()
         return x.loc[died_of_cause, "years_of_life_lost"].sum()
+
+    ##################
+    # Report methods #
+    ##################
+
+    def write_mortality_results(
+        self, cause: str, measure: str, results: pd.DataFrame
+    ) -> None:
+        measure_name = measure.split("_due_to_")[0]
+        write_dataframe_to_parquet(
+            results=results,
+            measure=measure_name,
+            entity_type="cause",
+            entity=cause,
+            sub_entity=None,  # FIXME
```

Comment:
Is this actually correct? Or can there be a subentity like implied in the [RT design doc (see second row)](https://vivarium-research.readthedocs.io/en/latest/model_design/vivarium_features/standard_results_formats/index.html#some-examples)

Reply:
@rmudambi But then what goes into entity?

Reply:
The RT design doc has the following example for deaths:
![image](https://github.com/ihmeuw/vivarium_public_health/assets/23350991/62ead7e6-304a-4647-a189-9cb094191bcc)

I feel like we're missing Disease information (i.e. "infected" state) at this point, though

Reply:
On wait - I think I see what I need to do